pkg/webui/console/containers/device-payload-formatters/downlink.js

```diff
@@ -100,6 +100,16 @@ class DevicePayloadFormatters extends React.PureComponent {
     })
   }
 
+  @bind
+  async onSubmitSuccess() {
```

Comment:
Please remove `L96-L100` then. Currently, two toasts are shown on successful payload formatter update.

Reply:
You can use the uplink form as a reference